readme.md

```diff
@@ -17,8 +17,13 @@ var splitLines = require('split-lines');
 
 splitLines('foo\r\nbar\r\nbaz\nrainbow');
 //=> ['foo', 'bar', 'baz', 'rainbow']
+
+splitLines('foo\r\nbar\r\nbaz\nrainbow', true);
+//=> ['foo\rn', 'bar\r\n', 'baz\n', 'rainbow']
 ```
 
+Not that the default behavior is to discard the separator characters. There is an optional second parameter `retainLineSeparators`, that causes the separators to be retained at the end of every line (except the last line, which will never contain a line separator).
```

Comment:
I would add an API section instead of this description. Something like this:

```
## API

### splitLines(input, [retainLineSeparators])

#### input

type: `string`

String to split.

#### retainLineSeparators

type: `boolean`<br>
default: `false`

Retain the separators and the end of every line. Except the last line which will never contain a line separator.
```